SU2_PY/SU2/io/historyMap.py

```diff
@@ -451,7 +451,7 @@
                 'GROUP': 'AERO_COEFF',
                 'HEADER': 'CEff',
                 'TYPE': 'COEFFICIENT'},
- 'EQUIV_AREA': {'DESCRIPTION': 'Equivalent area',
+ 'EQUIVALENT_AREA': {'DESCRIPTION': 'Equivalent area',
```

Comment:
Did you run the script (updateHistoryMap or something like that) to update this file or did it by hand?

Reply:
I updated it by hand. Now I know how to use updateHistoryMap.py, I used this script to update historyMap.py. It changed things that I didn't touch. Is it OK to update them?

Change
- BGS_TEMPERATURE

Added
- STREAMWISE_DP
- STREAMWISE_HEAT
- STREAMWISE_MASSFLOW

Reply:
It should be... @TobiKattmann I think these are "your" outputs, any objections?

Reply:
No objections, I forgot to do that in #773 obviously 😞 ... for bgs-temperature I changed the explaining string in the code which is automatically ported.

Reply:
OK, thanks.